.travis.yml

```diff
@@ -1,22 +1,26 @@
 language: php
 
 php:
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+  - hhvm
+
+matrix:
+  allow_failures:
+    - php: hhvm
+    - php: 7.0
 
 services:
-  - apc
   - memcache
-  - memcached # will start memcached
+  - memcached
 
 before_script:
   - composer update
```

Comment:
So this one could be argued either way.  Since our FW packages don't commit a `composer.lock` file, either one would truly be acceptable.  With that said though (and this is off-topic to your comment), we should probably look at testing dependencies on both the low and high ends of the spectrum using Composer's `--prefer-lowest` option (Symfony's [Travis-CI config](https://github.com/symfony/symfony/blob/v2.7.1/.travis.yml) has a good example of how they're testing this).